assets/js/force-2fa.js

```diff
@@ -0,0 +1,88 @@
+/* global ajaxurl, jQuery */
+
+/**
+ * Checks that an element has a non-empty `name` and `value` property.
+ *
+ * @param  {Element} element The element to check
+ * @return {Boolean} true if the element is an input, false if not
+ */
+var isValidElement = function( element ) {
```

Comment:
Should we wrap these methods in an anonymous function to isolate the namespace? Some of these function names are very generic and could cause conflicts.

Reply:
I'd really rather not - anonymous functions are a scourge to test and/or maintain and make the code far less readable. I would be happy to make a "namespace" via variable although the plugin would be better to add a build  pipeline from ES6 to address this kind of issue instead.